test_wikimedia_link_issue_reporter.py

```diff
@@ -4,7 +4,6 @@
 import wikibrain.wikimedia_link_issue_reporter
 import wikibrain
 import wikimedia_connection.wikimedia_connection as wikimedia_connection
-import wikimedia_connection.wikidata_processing as wikidata_processing
```

Comment:
wait, it was not needed?

Reply:
No, it is showing as not used in my editor

Reply:
and you have run tests in general, right?

Reply:
Yes of course, it is only used in the test_wikidata_structure